src/Trello/BoardManager.php

```diff
@@ -0,0 +1,63 @@
+<?php
+declare(strict_types=1);
+
+namespace AmsterdamPHP\TrelloChecklister\Trello;
+
+use AmsterdamPHP\TrelloChecklister\ExecutionResult;
+use DateTimeImmutable;
+use Stevenmaguire\Services\Trello\Client;
+use function array_filter;
+use const DATE_RFC3339;
+
+final class BoardManager
+{
+    /**
+     * @var Client
+     */
+    private $trello;
+
+    private $organizationId;
```

Comment:
Is `$organizationId` an object? A string? An integer? A bunch of bananas? This is a mistery!!

Reply:
PHP 7.4 pretty please